swift/ql/lib/codeql/swift/generated/expr/Argument.qll

```diff
@@ -3,7 +3,7 @@ import codeql.swift.elements.Element
 import codeql.swift.elements.expr.Expr
 
 class ArgumentBase extends @argument, Element {
-  override string toString() { result = "Argument" }
+  override string getPrimaryQlClass() { result = "Argument" }
```

Comment:
So where is `toString` now actually overridden for these classes?

Reply:
this is in the companion PR I'm writing, which implements it in the hand-written `Element` override

Reply:
Awesome. No complaints from me here then :)